File: CS/HowToCustomizeNewActionItemsList.Module/BusinessClasses.cs
using System;

using DevExpress.Xpo;

using DevExpress.ExpressApp;
using DevExpress.Persistent.Base;
using DevExpress.Persistent.BaseImpl;
using DevExpress.Persistent.Validation;

namespace HowToCustomizeNewActionItemsList.Module {
	[NavigationItem(GroupName="Group1")]
	[CreatableItem()]
	public class DomainObject1 : BaseObject {
		public DomainObject1(Session session)
			: base(session) {
		}
	}
	[NavigationItem(GroupName = "Group1")]
	[CreatableItem()]
	public class DomainObject2 : BaseObject {
		public DomainObject2(Session session)
			: base(session) {
		}
	}
	[NavigationItem(GroupName = "Group2")]
	[CreatableItem()]
	public class DomainObject3 : BaseObject {
		public DomainObject3(Session session)
			: base(session) {
		}
	}
	[NavigationItem(GroupName = "Group2")]
	[CreatableItem()]
	public class DomainObject4 : BaseObject {
		public DomainObject4(Session session)
			: base(session) {
		}
	}
}

File: VB/HowToCustomizeNewActionItemsList.Module/BusinessClasses.vb
Imports System

Imports DevExpress.Xpo

Imports DevExpress.ExpressApp
Imports DevExpress.Persistent.Base
Imports DevExpress.Persistent.BaseImpl
Imports DevExpress.Persistent.Validation

<NavigationItem(GroupName:="Group1"), CreatableItem()> _
Public Class DomainObject1
   Inherits BaseObject
   Public Sub New(ByVal session As Session)
      MyBase.New(session)
   End Sub
End Class
<NavigationItem(GroupName:="Group1"), CreatableItem()> _
Public Class DomainObject2
   Inherits BaseObject
   Public Sub New(ByVal session As Session)
      MyBase.New(session)
   End Sub
End Class
<NavigationItem(GroupName:="Group2"), CreatableItem()> _
Public Class DomainObject3
   Inherits BaseObject
   Public Sub New(ByVal session As Session)
      MyBase.New(session)
   End Sub
End Class
<NavigationItem(GroupName:="Group2"), CreatableItem()> _
Public Class DomainObject4
   Inherits BaseObject
   Public Sub New(ByVal session As Session)
      MyBase.New(session)
   End Sub
End Class

